test/IRGen/raw_layout_multifile.swift

```diff
@@ -1,7 +1,57 @@
-// RUN: %target-swift-frontend -enable-experimental-feature RawLayout -emit-ir -primary-file %s %S/Inputs/raw_layout_multifile_b.swift
-// RUN: %target-swift-frontend -enable-experimental-feature RawLayout -emit-ir %s -primary-file %S/Inputs/raw_layout_multifile_b.swift
-
-import Swift
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/chex.py < %s > %t/raw_layout_multifile.swift
+// RUN: %target-swift-frontend -enable-experimental-feature BuiltinModule -enable-experimental-feature RawLayout -emit-ir -O -disable-availability-checking %s %S/Inputs/raw_layout_multifile_b.swift | %FileCheck %t/raw_layout_multifile.swift --check-prefix=CHECK --check-prefix=CHECK-%target-ptrsize
 
 @_rawLayout(like: Int32)
 public struct Foo<T>: ~Copyable {}
+
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}5MyIntVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 4
+// stride
+// CHECK-SAME:  , {{i64|i32}} 4
+// flags: alignment 3, noncopyable
+// CHECK-SAME:  , <i32 0x800003>
+struct MyInt: ~Copyable {
+  let x: Int32Fake
+}
+
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}9BadBufferVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 48
+// stride
+// CHECK-SAME:  , {{i64|i32}} 48
+// flags: alignment 7, noncopyable, is not inline
+// CHECK-SAME:  , <i32 0x820007>
+struct BadBuffer: ~Copyable {
+  let buf = SmallVectorOf3<Int64?>()
+}
+
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}5WeirdVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 8
+// stride
+// CHECK-SAME:  , {{i64|i32}} 8
+// flags: alignment 7, noncopyable
+// CHECK-SAME:  , <i32 0x800007>
```

Comment:
@Azoy, this last check [fails on Android armv7](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android/5262/consoleText), does it pass on the 32-bit watchOS CI?
```
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/buildbot_linux/swift-linux-x86_64/test-android-armv7/IRGen/Output/raw_layout_multifile.swift.tmp/raw_layout_multifile.swift:36:16: error: CHECK-SAME: expected string not found in input
// CHECK-SAME: , i32 8388615
               ^
<stdin>:57:381: note: scanning from here
@"$s20raw_layout_multifile5WeirdVWV" = internal constant %swift.vwtable { ptr @__swift_cannot_copy_noncopyable_type, ptr @__swift_noop_void_return, ptr @__swift_cannot_copy_noncopyable_type, ptr @__swift_cannot_copy_noncopyable_type, ptr @__swift_memcpy8_8, ptr @__swift_memcpy8_8, ptr @"$s20raw_layout_multifile5WeirdVwet", ptr @"$s20raw_layout_multifile5WeirdVwst", i32 8, i32 8, i32 8519687, i32 0 }, align 4
                                                                                                                                                                                                                                                                                                                                                                                            ^
```